chromadb/utils/embedding_functions/roboflow_embedding_function.py

```diff
@@ -29,7 +31,7 @@ def __init__(self, api_key: str = "", api_url="https://infer.roboflow.com") -> N
             api_url (str, optional): The URL of the Roboflow API. Defaults to "https://infer.roboflow.com".
         """
         if not api_key:
-            api_key = os.environ.get("ROBOFLOW_API_KEY")
```

Comment:
`None` is incompatible with the `str` type of the signature. I don't see much value in this conditional logic as given the default, a `not api_key` would mean the user passing an empty string on purpose which I find quite unlikely, so happy to remove this bit altogether (maybe as a part of the follow up PR)